pages/login/index.js

```diff
@@ -0,0 +1,234 @@
+import { SERVICE_HOSTNAME, SIGNIN_USER_ENDPOINT, EXTENSION_ID, GOOGLE_LOGIN, PASSWORD_LOGIN, SIGNUP_USER_ENDPOINT } from "../../cfg/endpoints.js";
+import { MESSAGE_TYPES } from "../../cfg/messages.js";
+import { createNode } from "../../lib/user-agent.js";
+import { logInfo } from "../../lib/core.js";
+import { getWidth } from "../../lib/graphics.js"
+import { simpleFetchAndCheck } from "../../lib/network.js";
+
+let _extensionId = 'dpkajdodpdckkjhhdnpgmimdmceeilkc' //EXTENSION_ID
+let _forceServicesHostname = null
+const _LOG_SCOPE = '[Trellus][External signin]'
+
+let _is_signin_screen = true
+add_signin_form()
+
+// add envent listener for click
+document.querySelector('#sign-in-type-button').addEventListener('click', () => updateLogin())
+
+function add_signin_form() {
+
+  const formDiv = document.querySelector('#form')
+  formDiv.innerHTML = ''
+
+  const emailInput = createNode("input", {"type": "email", "id": "email", "placeholder": "Email", "class": "inputField", "contenteditable": true, "data-field": "email"})
+  const passwordInput = createNode("input", {"type": "password", "id": "password", "style": "margin-top: 10px;", "placeholder": "Password", "class": "inputField", "contenteditable": true, "data-field": "password"})
+
+  const signinbutton = createNode("div", {"class": "button ctaPrimary", "style": "margin-top: 10px", "id": "sign-in-button"}, "Sign in")
```

Comment:
so the two ways this is typically done:
1)  use <template> html tags
2) put both the signup and signin form in the doc, but then use JS to control which is hidden